src/components/FormComponents/Pasientopplysninger/Adresser/Utenlandskadresse.tsx

```diff
@@ -0,0 +1,24 @@
+import { BodyShort } from '@navikt/ds-react';
+
+import { UtenlandskAdresse } from '../../../../graphql/queries/graphql.generated';
+import { checkValue } from '../../../../utils/formUtils';
+import countries from '../../../../pages/api/country/countries-norwegian.json';
+
+type UtenlandskAdresseProps = {
+    utenlandskAdresse: UtenlandskAdresse;
+};
+
+function UtenlandskAdresse({ utenlandskAdresse }: UtenlandskAdresseProps): JSX.Element {
+    const country = countries.find((country) => country.alpha3.toUpperCase() === utenlandskAdresse.landkode);
+
+    return (
+        <>
+            <BodyShort>{checkValue(utenlandskAdresse.adressenavnNummer)}</BodyShort>
+            <BodyShort>{checkValue(utenlandskAdresse.postboksNummerNavn)}</BodyShort>
+            <BodyShort>{`${checkValue(utenlandskAdresse.postkode)} ${checkValue(utenlandskAdresse.bySted)}`}</BodyShort>
+            <BodyShort>{country?.name}</BodyShort>
```

Comment:
ref. kommentaren lengre ned, så kunne det kanskje vært

```suggestion
            <BodyShort>{utenlandskAdresse.adressenavnNummer ?? ''}</BodyShort>
            <BodyShort>{utenlandskAdresse.postboksNummerNavn ?? '}</BodyShort>
            <BodyShort>{`${utenlandskAdresse.postkode ?? ''} ${utenlandskAdresse.bySted ?? ''}`}</BodyShort>
            <BodyShort>{country?.name}</BodyShort>
```

Men siden om de er tomme stringer så blir det tomme divs, burde man heller rendret de conditionally?